.gitignore

```diff
@@ -80,6 +80,9 @@ man/ctags-incompatibilities.7.rst
 man/ctags-optlib.7
 man/ctags-optlib.7.html
 man/ctags-optlib.7.rst
+man/tags.5
+man/tags.5.html
+man/tags.5.rst
```

Comment:
tags.5.rst was added to repository.
Unlike the other man pages, tags.5.rst is not generated from tags.5.rst.in.
So man/tags.5.rst should not be ignored.

Reply:
Oh, I will fix it.